AnkiDroid/src/main/java/com/ichi2/ui/ActionBarOverflow.java

```diff
@@ -113,11 +115,11 @@ static boolean hasUsableMethod() {
 
     @CheckResult
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
-    static Pair<Class, Method> getPrivateMethodOnlyHandleExceptions(String className, String methodName) {
+    static Pair<Class<?>, Method> getPrivateMethodOnlyHandleExceptions(String className, String methodName) {
         Method action = null;
         Class<?> menuItemImpl = null;
         try {
-            menuItemImpl = Class.forName(className);
+            menuItemImpl = forName(className);
```

Comment:
I don't think this has value
```suggestion
            menuItemImpl = Class.forName(className);
```